### ISSUE TITLE
Display menu size changes depending on net unit

### ISSUE THREAD

**Issue by cbcoutinho** [paradoxxxzero/gnome-shell-system-monitor-applet#438]
I have the extension installed on my OpenSUSE Tumbleweed laptop through gnome.extensions.org. I noticed today that the dropdown window flickers slightly when the internet speed jumps between units. Specifically, when my download speed goes from `_ kbit/s Down` to `_ Mbit/s Down`, the horizontal spacing increase causes the window to increase accordingly.

I know it isn't terrible, but when the speed is hovering around ~1 Mbit/s Down, the window looks like it's flickering:

![output](https://user-images.githubusercontent.com/12901868/37374383-2fef9efc-271b-11e8-97c8-8457b10c4884.gif)

```shell
$ gnome-shell --version
GNOME Shell 3.26.2

$ cat ~/.local/share/gnome-shell/extensions/system-monitor@paradoxxx.zero.gmail.com/metadata.json 
{
  "_generated": "Generated by SweetTooth, do not edit", 
  "description": "Display system informations in gnome shell status bar, such as memory usage, cpu usage, network rates\u2026", 
  "gettext-domain": "system-monitor", 
  "name": "system-monitor", 
  "settings-schema": "org.gnome.shell.extensions.system-monitor", 
  "shell-version": [
    "3.4", 
    "3.6", 
    "3.8", 
    "3.10", 
    "3.12", 
    "3.14", 
    "3.16", 
    "3.18", 
    "3.20", 
    "3.22", 
    "3.24", 
    "3.26"
  ], 
  "url": "https://github.com/paradoxxxzero/gnome-shell-system-monitor-applet", 
  "uuid": "system-monitor@paradoxxx.zero.gmail.com", 
  "version": 34
```

**Comment by piegamesde** [comment]
This is a known issue #434 and it is equally annoying to my. I'll try to fix it once my current changes are merged into master

**Comment by franglais125** [comment]
Here is a possible solution : https://github.com/franglais125/gnome-shell-system-monitor-applet/tree/fix_width

It uses a monospace font for the units, so that the width is constant. Would you mind testing it?

**Comment by piegamesde** [comment]
@franglais125 With the fix, here's what works:
- In the system tray, the frequency digit does not change its size anymore

What doesn't:
- The frequency display in the menu still changes its width when jumping above >1GHz
- The network traffic display in the menu does technically work, but the values are zero-padded and that doesn't look very good IMO.

**Comment by franglais125** [comment]
@piegamesde Would you mind trying again with that branch? I made a few changes.

**Comment by piegamesde** [comment]
It works fine now.

**Comment by franglais125** [comment]
Thanks for the input. Fixed in master [a53d3cb43ba9c3c4070e75335fa21ae1a48aca01].
